proposals/lip-0044.md

```diff
@@ -511,46 +545,13 @@ def getAddressFromBLSKey(blsKey: PublicKeyBLS) -> Address:
     return registeredBLSKeys(blsKey)
 ```
 
-#### getGeneratorList
-
-This function returns the current generator list `generatorList`.
-
-```python
-def getGeneratorList() -> list[Address]:
-    return generatorList
-```
-
-#### getSlotNumber
+#### getValidatorParams
```

Comment:
This is introduced so that other modules have the flexibility to obtain this information from the Validators module. However, for our modules we don't need it so far.